tiproxy/tiproxy-api.md

```diff
@@ -0,0 +1,128 @@
+---
+title: TiProxy API
+summary: Learn how to use the TiProxy API to access its configuration, health status, and monitoring data.
+---
+
+# TiProxy API
+
+[TiProxy](/tiproxy/tiproxy-overview.md) provides API endpoints for accessing its configuration, health status, and monitoring data.
+
+> **Note:**
+>
+> TiProxy API is specifically designed for debugging purposes and might not be fully compatible with future capabilities introduced in TiProxy. It's not recommended to include this tool in applications or utilities development to get information.
+
+The address for accessing the TiProxy API is `http://${host}:${port}`, where `host` and `port` are specified by the TiProxy configuration item [`api.addr`](/tiproxy/tiproxy-configuration.md#addr-1). For example:
+
+```bash
+curl http://127.0.0.1:3080/api/admin/config/
+```
+
+## Get TiProxy configuration
+
+### Request URI
+
+`GET /api/admin/config/`
+
+### Parameter descriptions
+
+The query parameter is as follows:
+
+- `format`: (optional) specifies the format of the returned configuration. Value options are `json` and `toml`. The default value is `toml`.
+
+### Example
+
+The following example gets the TiProxy configuration in JSON format:
+
+```bash
+curl "http://127.0.0.1:3080/api/admin/config/?format=json"
+```
+
+## Set TiProxy configuration
+
+This endpoint is used to modify the TiProxy configuration using the TOML format. Unspecified configuration items will remain unchanged, so you only need to specify the items that you want to modify.
```

Comment:
there is a comment for this paragraph in the Chinese PR
https://github.com/pingcap/docs-cn/pull/17340/files#r1619712746